internal/action/actions.go

```diff
@@ -195,6 +206,16 @@ func (h *BufPane) MoveCursorDown(n int) {
 			vloc.SLoc = sloc
 			vloc.VisualX = h.Cursor.LastVisualX
 			h.Cursor.Loc = h.LocFromVLoc(vloc)
+			if util.IntOpt(h.Buf.Settings["wrapindent"]) > -1 {
+				movingtorow := vloc.SLoc.Row
```

Comment:
We don't need to store it as a variable which is just to be in a condition on the next line, unless I am missing something here.

Reply:
Thanks, you're right - the named variable was just for readability to make it clear which row was being referred to, but happy to remove this and clarify in the comment instead.